File: Languages/Golang/Screen.go
package main

import (
	"fmt"
)

// Golang uses structure instead of class
// What is the difference between structure and class?

type Screen struct{}

func (s *Screen) DisplayMessage(message string) {
	fmt.Println(message)
}

func (s *Screen) DisplayMessageWithNewLine(message string) {
	fmt.Println(message)
}

func (s *Screen) DisplayDollarAmount(amount float64) {
	fmt.Printf("$%.2f", amount)
}

File: Languages/Java/Screen.java
package Languages.Java;


public class Screen {
    // method to display message 
    // void means it returns nothing 
    public void displayMessage(String message) {
        System.out.println(message); 
    }

    // method to display message line
    public void displayMessageLine(String message) {
        System.out.println(message); 
    }

    // display dollar amount 
    public void displayDollarAmount(double amount) {
        System.out.printf("$%,.2f", amount);
    }
}